di-ipv-orchestrator-stub/src/main/java/uk/gov/di/ipv/stub/orc/config/OrchestratorConfig.java

```diff
@@ -16,8 +16,12 @@ public class OrchestratorConfig {
             getConfigValue("ORCHESTRATOR_CLIENT_ID", "di-ipv-orchestrator-stub");
     public static final String ORCHESTRATOR_REDIRECT_URL =
             getConfigValue("ORCHESTRATOR_REDIRECT_URL", "http://localhost:8083/callback");
+    public static final String ORCHESTRATOR_JAR_SIGNING_KEY =
+            getConfigValue("ORCHESTRATOR_JAR_SIGNING_KEY", "missing-jar-signing-key");
```

Comment:
In fact I think we should make a change here. We won't need another signing key. Ideally we should just use the one below on line 21 instead and just update that certificate to be the correct format

Reply:
We'd need to make a minor change somewhere else in the orc stub to make sure that we are using EC for all of our signing. But we can talk about that tomorrow